includes/Modules/Thank_With_Google.php

```diff
@@ -245,8 +247,42 @@ protected function create_data_request( Data_Request $data ) {
 				$service = $this->get_service( 'subscribewithgoogle' );
 				/* @var $service Google_Service_SubscribewithGoogle phpcs:ignore Squiz.PHP.CommentedOutCode.Found */
 
-				/* @TODO filter by verified domains */
-				return $service->publications->listPublications();
+				$sc_settings    = $this->options->get( Search_Console_Settings::OPTION );
+				$sc_property_id = $sc_settings['propertyID'];
+				$url_normalizer = new Google_URL_Normalizer();
+				$raw_url        = $url_normalizer->normalize_url(
```

Comment:
Using the normalizer here shouldn't be necessary because the host and URL permutation methods account for UTF-8+punycode variants. It probably doesn't hurt but shouldn't be needed.

Reply:
Thank you for the explanation, @aaemnnosttv! I've removed it.